cmd/list.go

```diff
@@ -80,12 +82,25 @@ func doList(cmd *cobra.Command, args []string) error {
 				} else {
 					v = strconv.Quote(string(value))
 				}
-
-				fmt.Fprintln(w, strings.Join([]string{secret.Name, string(secret.Type), key, v}, "\t"))
+				formattedSecrets = append(formattedSecrets, []string{secret.Name, string(secret.Type), key, v})
 			}
 		}
 	}
 
+	// sort by KEY
+	sort.Slice(formattedSecrets, func(i, j int) bool {
+		return formattedSecrets[i][2] < formattedSecrets[j][2]
+	})
+
+	// sort by NAME
+	sort.SliceStable(formattedSecrets, func(i, j int) bool {
+		return formattedSecrets[i][0] < formattedSecrets[j][0]
+	})
+
+	for _, secret := range formattedSecrets {
+		fmt.Fprintln(w, strings.Join(secret, "\t"))
+	}
```

Comment:
Because I didn't want to duplicate the "sort" and "print" processes.
I'll fix it.